mobile-widgets/qml/HintsTextEdit.qml

```diff
@@ -14,7 +14,11 @@ TextField {
 	onTextChanged: {
 		textUpdateTimer.restart();
 	}
-	onFocusChanged: frame.shouldShow = focus
+	onFocusChanged: {
+		focus != focus
```

Comment:
Precisely, anyway I rebased the commits since that part was wrong.
It was probably an conflict with the position, changing  focus on focus changing, generated a loop that caused the app to crash.